slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,229 @@
+-   Package is part of the mission, but to emphasize our goal is to push out information on practices for engineering high-quality open-source statistical software
+
+## Long term perspective
+
+-   Software engineering is a critical competence in producing high-quality statistical software
+-   A lot of work need to be done regarding the establishment, dissemination and adoption of best practices for engineering open-source software
+-   Improving the way software engineering is done will help improve the efficiency, reliability and innovation within Biostatistics
+
+## Comparing SAS and R
+
+To run an mmrm model in SAS it is recommended to use either the PROC MIXED or PROC GLM procedures. 
+
+- Less model assumptions are applied in PROC MIXED, primarily how one treats missingness.
+
+- We will compares the PROC MIXED procedure to the `{mmrm}` package in the following attributes:
+
+::: columns
+::: {.column width="50%"}
+- Documentation
+- Unit Testing
+- Estimation Methods
+:::
+::: {.column width="50%"}
+- Covariance structures
+- Degrees of Freedom
+- Contrasts
+:::
+:::
+
+## Documentation
+
+Both languages have online documentation of the technical details of the estimation and degrees of freedom methods and the different covariance structures available.
+
+::: columns
+::: {.column width="50%"}
+`{mmrm}`
+:::
+::: {.column width="50%"}
+`PROC MIXED`
+:::
+:::
+
+::: columns
+::: {.column width="50%"}
+- [Homepage](https://openpharma.github.io/mmrm/main/index.html)
+- [Usage](https://openpharma.github.io/mmrm/main/articles/introduction.html)
+- [Estimation](https://openpharma.github.io/mmrm/main/articles/algorithm.html)
+- [Covariance Structures](https://openpharma.github.io/mmrm/main/articles/covariance.html)
+- Degrees of Freedom [KR](https://openpharma.github.io/mmrm/main/articles/kenward.html)/[Satterthwaite](https://openpharma.github.io/mmrm/main/articles/satterthwaite.html)
+:::
+::: {.column width="50%"}
+- [Homepage](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.4/statug/statug_mixed_toc.htm)
+- [Usage](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.4/statug/statug_mixed_examples02.htm)
+- [Theory](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.4/statug/statug_mixed_details01.htm)
+- [Covariance Structures](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.4/statug/statug_mixed_syntax14.htm#statug.mixed.repeatedstmt_type)
+- [Degrees of Freedom](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.4/statug/statug_mixed_syntax10.htm#statug.mixed.modelstmt_ddfm)
+:::
+:::
+
+## Unit Testing
+
+One major advantage of the `{mmrm}` over `PROC MIXED` is that the unit testing in `{mmrm}` is transparent. It uses the `{testthat}` framework with `{covr}` to communicate the testing coverage. Unit tests can be found in the GitHub repository under [./tests](https://github.com/openpharma/mmrm/tree/main/tests/testthat).
```

Comment:
As discussed yesterday, I would put here something around integration tests comparing to SAS results, with typical relative tolerance of 10^-3